Code/GraphMol/MMPA/Wrap/rdMMPA.cpp

```diff
@@ -47,6 +48,116 @@ python::tuple fragmentMolHelper(const RDKit::ROMol& mol, unsigned int maxCuts,
   }
   return python::tuple(pyres);
 }
+
+python::tuple fragmentMolHelper2(const RDKit::ROMol& mol,
+                                 unsigned int minCuts,
+                                 unsigned int maxCuts,
+                                 unsigned int maxCutBonds,
+                                 const std::string& pattern,
+                                 bool resultsAsMols) {
+  std::vector<std::pair<RDKit::ROMOL_SPTR, RDKit::ROMOL_SPTR> > tres;
+  bool ok = RDKit::MMPA::fragmentMol(mol, tres, minCuts, maxCuts, maxCutBonds,
+                                     pattern);
+  python::list pyres;
+  if (ok) {
+    for (std::vector<std::pair<RDKit::ROMOL_SPTR,
+                               RDKit::ROMOL_SPTR> >::const_iterator pr =
+             tres.begin();
+         pr != tres.end(); ++pr) {
+      python::list lres;
+      if (resultsAsMols) {
+        lres.append(pr->first);
+        lres.append(pr->second);
+      } else {
+        if (pr->first) {
+          lres.append(RDKit::MolToSmiles(*(pr->first), true));
+        } else {
+          lres.append("");
+        }
+        lres.append(RDKit::MolToSmiles(*(pr->second), true));
+      }
+      pyres.append(python::tuple(lres));
+    }
+  }
+  return python::tuple(pyres);
+}
+
+template<class T>
```

Comment:
Seems like this does the same thing as `pythonObjectToVect()` in `RDBoost/Wrap.h`